src/main/kotlin/no/nav/tiltakspenger/saksbehandling/behandling/domene/Behandling.kt

```diff
@@ -69,6 +69,7 @@ data class Behandling(
     val barnetillegg: Barnetillegg?,
     val valgteTiltaksdeltakelser: ValgteTiltaksdeltakelser?,
     val avbrutt: Avbrutt?,
+    val antallDagerPerMeldeperiode: Int?,
```

Comment:
Jeg trodde vi hadde en type for dette, men finner den ikke. Den kan jo sikre at denne er mellom 1 og 14 f.eks.

Reply:
Jeg kan jo legge på en require for dette :)